Xamarin.Forms.Platform.iOS/Renderers/ScrollViewRenderer.cs

```diff
@@ -208,6 +208,25 @@ void OnScrollToRequested(object sender, ScrollToRequestedEventArgs e)
 			else
 			{
 				var positionOnScroll = Controller.GetScrollPositionForElement(e.Element as VisualElement, e.Position);
+
+				if (positionOnScroll.X < 0)
+				{
+					positionOnScroll.X = 0;
+				}
+				else if (positionOnScroll.X > ContentSize.Width - Bounds.Size.Width)
+				{
+					positionOnScroll.X = ContentSize.Width - Bounds.Size.Width;
+				}
+
+				if (positionOnScroll.Y < 0)
+				{
+					positionOnScroll.Y = 0;
+				}
+				else if (positionOnScroll.Y > ContentSize.Height - Bounds.Size.Height)
+				{
+					positionOnScroll.Y = ContentSize.Height - Bounds.Size.Height;
+				}
```

Comment:
could you use Math.Min() and Math.Max() for this ?

`positionOnScroll.X = Math.Max(positionOnScroll.X, 0);` that'd help the readability